docs/_content/standaard/verzoeken/index.md

```diff
@@ -0,0 +1,69 @@
+# Inleiding
+Een verzoek is een aanvraag of opdracht aan de gemeente (of andere overheid) voor de levering van een product of dienst. Verzoeken vormen dus de schakel tussen de klant, producten en diensten van de gemeente zoals in de PDC staan, en het zaakgericht werken voor de afhandeling. Het concept verzoeken is voor het eerst geintroduceerd in GFO Zaken, maar had in het opvolgende informatiemodel RGBZ geen plaats. In de visie van VNG Realisatie is dat echter een omissie (redenen zijn hieronder gegeven), en daarom is er behoefte aan een concept dat vooraf gaat aan de daadwerkelijke behandeling in een zaak (of andere activiteit).
+
+Met verzoeken introduceren we een nieuw concept in het zaakgericht werken. Voor het bepalen van de functionaliteit die een eerste implementatie van de Verzoeken API moet bieden, gaan we uit van de ingediende user stories en andere/eerdere implementaties van het begrip 'verzoeken' (zie de bronnenlijst onderaan). Op basis daarvan kan een eerste set ontwerpbeslissingen en uitgangspunten worden opgesteld. Die zijn in dit document beschreven. Tegelijkertijd zijn over de precieze verhouding tussen 'verzoeken' en een aantal andere concepten in het zaakgericht werken, met name 'zaak' en '(klant)contactmoment', nog een aantal vragen te beantwoorden (zie relaties/kardinaliteiten hieronder). Mogelijk wordt de Verzoeken API naar aanleiding daarvan in de toekomst uitgebreid. 
+
+# Redenen voor het ontwikkelen van een Verzoeken API
+Het (her)introduceren van verzoeken in het zaakgericht werken biedt een aantal voordelen, onder andere op de volgende gebieden:
+* Compleetheid: we standaardiseren een extra deel van het proces dat leidt tot levering van producten en diensten, namelijk (het generieke deel van) de gegevens die nodig zijn voor het aanvragen hiervan. Hiermee kunnen we (het generieke deel van) StUF-EF vervangen.
+* Herkenbaarheid: de klant communiceert met de gemeente niet langer over zaken (een voor hem niet altijd herkenbaar concept), maar over verzoeken, die (als verzoeken correct worden gebruikt) een voor de klant herkenbare (combinatie van) product(en) vertegenwoordigen.
```

Comment:
Als het verzoek leidt tot een zaak dan wil de klant ook daarna de zaak willen volgen door middel van (zaak)statussen, lijkt me.  Dus het verzoekconcept standaardiseert alleen de statussen die doorlopen worden bij de intake, maar dat is alleen het eerste stukje van de afhandeling.